main_routers/pages_router.py

```
@@ -0,0 +1,96 @@
+
+
+@router.get('/memory_browser', response_class=HTMLResponse)
+async def memory_browser(request: Request):
+    templates = get_templates()
+    return templates.TemplateResponse("templates/memory_browser.html", {"request": request})
+
+
+# IMPORTANT: This catch-all route MUST be registered LAST
+# to avoid matching static paths like /l2d, /api_key, /memory_browser, etc.
+@router.get("/{lanlan_name}", response_class=HTMLResponse)
+async def get_index(request: Request, lanlan_name: str):
```

Ruff (0.14.8)

Unused function argument: `lanlan_name`

(ARG001)